### PR TITLE
Add a template for java-jbang

### PR DESCRIPTION
**Pull request by maxandersen**
matching with adding jbang to pulumi-language-java.

See https://github.com/pulumi/pulumi-java/pull/575

### CODE REVIEW

**Comment by maxandersen** [comment]
Can someone /run-templates ?

**Comment by t0yv0** [comment]
Waiting on https://github.com/pulumi/pulumi-java/pull/575 to be released in the next version of pulumi.

**Comment by maxandersen** [comment]
Sounds good. Just heads up I'm on PTO next week so just be aware I might not react before week after :)

**Comment by maxandersen** [comment]
Now jbang support is Released anything blocking this from being merged?

**Review by t0yv0 — APPROVED** [review]
Tested and it works with pulumi v3.34.0

File: java-jbang/src/main.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+//DEPS com.pulumi:pulumi:0.1.0
```
Comment (t0yv0) [inline java-jbang/src/main.java:1]: Could we make it a floating ref?

Comment (t0yv0) [reply java-jbang/src/main.java:1]: Perhaps `//DEPS com.pulumi:pulumi:0.+` - this seems to work for me.

**Comment by stack72** [comment]
Please view the results of the PR Build [Here][1]

[1]: https://github.com/pulumi/templates/actions/runs/2471226344

File: java-jbang/src/main.java (hunk 1 above)
Comment (maxandersen) [reply java-jbang/src/main.java:1]: yes, that syntax kinda works. probably will add "(,1.0]" syntax in future.

**Comment by t0yv0** [comment]
Merged via https://github.com/pulumi/templates/pull/297 thank you!

**Pull request closed without merging**